src/utils/hints/accountHints.js

```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
+  hash: 'Transaction hash of transaction sent by account.',
+  time: 'Estimated date, time, and block height when the transaction has been sent from the account.',
+  type: 'Type of the transaction sent by account.',
+  name: 'Name owned by this account.',
+  expires: 'Estimated date, time, and block height when the name owned by the account expires.',
```

Comment:
I might be wrong but If I read this it seems that the block height is also estimated. I would maybe change it to:
"Block height and estimated date when..."

Reply:
fixed